apps/frontend/src/pages/CreateAccount.tsx

```diff
@@ -1,5 +1,172 @@
+      return;
+    } else {
+      // Clear email error if valid
+      setEmailError("");
+    }
+
+    if (!passwordFormat.test(password)) {
+      setPasswordError("Password does not meet the requirements.");
+      return;
+    } else {
+      // Clear error message if password is valid
+      setPasswordError("");
+    }
+
+    try {
+      const response = await axios.post("/api/user", {
+        username,
+        name: username, // Assuming 'name' is required as same as 'username'
+        email,
+        password,
+      });
+      if (response.status === 201) {
+        console.log("User created successfully", response.data);
+        // Perform any actions after successful account creation, like redirecting to login page
+      }
+    } catch (error) {
+      console.error("Failed to create account", error);
+      // Handle server errors here
+      setPasswordError("An unexpected error occurred");
+    }
+
+    // Temporary navigation to homepage until backend is ready: ticket #141
+    navigate("/");
+  };
+
+  return (
+    <Box sx={classes.loginAndCreateBox}>
+      <Box sx={{ mb: 2 }}>
+        <img
+          src={martletPlaceLogo}
+          alt="MartletPlace Logo"
+          style={{ width: "100px", marginBottom: "16px" }}
+        />
+      </Box>
+      <Typography variant="h4" component="h1" gutterBottom>
+        University of Victoria
+      </Typography>
+      <Typography variant="h4" component="h1" gutterBottom>
+        MartletPlace
+      </Typography>
+      <Box component="form" onSubmit={handleCreateAccount} sx={classes.form}>
+        <TextField
+          label="Email, must be a valid UVic email"
+          type="email"
+          variant="outlined"
+          required
+          fullWidth
+          margin="normal"
+          value={email}
+          onChange={(e) => setEmail(e.target.value)}
+          error={!!emailError}
+        />
+        {emailError && <FormHelperText error>{emailError}</FormHelperText>}
+
+        <TextField
+          label="Username"
+          variant="outlined"
+          required
+          fullWidth
+          margin="normal"
+          value={username}
+          onChange={(e) => setUsername(e.target.value)}
+        />
+        <TextField
+          label="Password"
+          variant="outlined"
+          required
+          type="password"
+          fullWidth
+          margin="normal"
+          value={password}
+          onChange={(e) => setPassword(e.target.value)}
+          error={!!passwordError}
+        />
+        {passwordError && (
+          <FormHelperText error>{passwordError}</FormHelperText>
+        )}
+        <Typography
+          variant="body2"
+          color={colors.martletplaceGrey}
+          gutterBottom
+          sx={{
+            width: "100%",
+            textAlign: "left",
+            mt: 1,
+          }}
+        >
+          Password must have at least:
+          <br />
+          8 characters
+          <br />
```

Comment:
<br /> should generally be avoided and css should be used for spacing instead. There's nothing inherently wrong with using them, it's just generally not considered semantic unless there's a specific reason for the breaks (most common example being displaying poetry)